cmd/addblock/config.go

```diff
@@ -79,6 +80,7 @@ func netName(chainParams *chaincfg.Params) string {
 	switch chainParams.Net {
 	case wire.TestNet3:
 		return "testnet"
+	// TODO do we need to use "testnet" dir for testnet4 as well?
```

Comment:
We should use a distinct directory, otherwise you couldn't run both on the same computer.